csrankings-h.csv

```diff
@@ -759,6 +759,7 @@ Hua Huang 0003,Univ. of California - Merced,https://sites.ucmerced.edu/huahuang,
 Hua Lu 0001,Aalborg University,http://people.cs.aau.dk/~luhua,h1mExmcAAAAJ
 Hua Ming,Oakland University,http://www.secs.oakland.edu/~ming/publications,NOSCHOLARPAGE
 Hua Wang 0007,Colorado School of Mines,http://inside.mines.edu/~huawang,TZ5nsRUAAAAJ
+Huajie Shao, College of William and Mary,https://shj1987.github.io/,5-D7ZLsAAAAJ&hl=en

```

Comment:
Remove spurious space after comma and remove `&hl=en` from Google Scholar ID.

Reply:
I have fixed the issue now. Thanks a lot!